docs/spec/TEMPLATE.md

```diff
@@ -0,0 +1,67 @@
+# ConceptName
+
+<!--
+Enter an English description of the concept here and a rough explanation of how
+it's mapped to the Kubernetes model.
+-->
+
+ConceptName in CF is mapped to Kubernetes BackingObjects.
+
+<!--
+Populate this table with the action available on the Cloud controller in the
+Action column and the Kubernetes verb and noun in the right hand column.
+
```

Comment:
Below you use a special string `_not_mapped_` can you use that as an example or describe it in this comment?